packages/ppx_string_interpolation/ppx_string_interpolation.1.0.0/opam

```diff
@@ -0,0 +1,30 @@
+opam-version: "2.0"
+name: "ppx_string_interpolation"
+synopsis: "String interpolation PPX preprocessor"
+description: "
+String interpolation PPX preprocessor.
+"
+maintainer: "Konstantin Romanov <konstantinsromanov@gmail.com>"
+authors: ["Konstantin Romanov <konstantinsromanov@gmail.com>"]
+homepage: "https://https://github.com/bloomberg/ppx_string_interpolation"
+bug-reports: "https://https://github.com/bloomberg/ppx_string_interpolation/issues"
+doc: "https://bloomberg.github.io/ppx_string_interpolation/README.md"
+dev-repo: "git+https://github.com/bloomberg/ppx_string_interpolation.git"
+license: "Apache-2.0"
+
+url {
+  src: "https://github.com/bloomberg/ppx_string_interpolation/archive/1.0.0.tar.gz"
+    checksum: "md5=3e65a57fd7fb6c7025e7ade74b22c83a"
+    }
+
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+]
+
+depends: [
+  "dune" {build}
+  "ocaml" {>= "4.04.1"}
+  "ppxlib" {build}
```

Comment:
```suggestion
  "ppxlib"
```
This package seems to be used in a library. Libraries need to be rebuilt when one of their dependencies change

Reply:
Thanks, addressed.